src/github/comments/performanceReportComment.ts

```diff
@@ -0,0 +1,40 @@
+import {PerformanceReport} from "../../types.js";
+import {renderPerformanceTable} from "../../utils/render.js";
+
+export function performanceReportComment(report: PerformanceReport): string {
+  const isFailedResults = report.results.filter((r) => r.isFailed);
+  const isImprovedResults = report.results.filter((r) => r.isImproved);
+
+  if (report.someFailed) {
+    return `
+## :warning: **Performance Alert** :warning:
+
+Possible performance regression was detected for some benchmarks.
+Benchmark result of this commit is worse than the previous benchmark result exceeding threshold.
+
+${renderPerformanceTable(isFailedResults, report, "html")}
+`;
+  }
+
+  if (isImprovedResults.length > 0) {
+    return `
+## Performance Report
+
+🚀🚀 Significant benchmark improvement detected
+
+<details><summary>Full benchmark results</summary>
+${renderPerformanceTable(isImprovedResults, report, "html")}
```

Comment:
how was this comment being generated before? where was the text coming from?

This looks like a bug here where only the improved results are shown, rather than the "Full benchmark results".

Reply:
Fixed now.